x/dogfood/keeper/keeper.go

```diff
@@ -114,3 +111,24 @@ func (k Keeper) mustValidateFields() {
 	types.PanicIfNil(k.delegationKeeper, "delegationKeeper")
 	types.PanicIfNil(k.restakingKeeper, "restakingKeeper")
 }
+
+// Add the function to get detail information through the operatorKeeper within the dogfood
+func (k Keeper) ValidatorByConsAddrForChainID(ctx sdk.Context, consAddr sdk.ConsAddress, chainID string) (stakingtypes.Validator, bool) {
+	return k.operatorKeeper.ValidatorByConsAddrForChainID(ctx, consAddr, chainID)
+}
+
+func (k *Keeper) GetStakersByOperator(ctx sdk.Context, operator, assetID string) (delegationtype.StakerList, error) {
```

Comment:
only this function is  pointer receiver, is it specail ?

Reply:
For compatible with other modules